server/server/src/main.rs

```diff
@@ -1,25 +1,70 @@
 #![allow(where_clauses_object_safety)]
 
-use std::env;
-
+use fast_log::{
+    consts::LogSize,
+    plugin::{file_split::RollingType, packer::LogPacker},
+    Config,
+};
+use log::LevelFilter;
 use server::{configuration, start_server};
-use service::settings::Settings;
+use service::settings::{LogDestination, LoggingSettings, Settings};
+use std::{env, fs::create_dir_all};
 use tokio::sync::oneshot;
 
 #[actix_web::main]
 async fn main() -> std::io::Result<()> {
-    if env::var("RUST_LOG").is_err() {
-        //Default rust log level to info
-        env::set_var("RUST_LOG", "info");
-    }
-    env_logger::init();
-
     let settings: Settings =
         configuration::get_configuration().expect("Failed to parse configuration settings");
+    logging_init(&settings.logging);
 
     let (off_switch, off_switch_receiver) = oneshot::channel();
     let result = start_server(settings, off_switch_receiver).await;
     // off_switch is not needed but we need to keep it alive to prevent it from firing
     let _ = off_switch;
     result
 }
+
+fn logging_init(settings: &LoggingSettings) {
```

Comment:
should probably be `initialise_logging` but I liked thinking of this in a cockney accent.. "it's loggin, innit??"
Could also move this out to a separate file, but thought that it was ok to keep in main 🤷